components/homme/src/share/control_mod.F90

```diff
@@ -617,4 +632,73 @@ subroutine test_timestep_make_parameters_consistent(par, nerr)
          write(iulog,'(a,i2)') 'test_timestep_make_parameters_consistent nerr', nerr
   end subroutine test_timestep_make_parameters_consistent
 
+
+subroutine set_planar_defaults()
+
```

Comment:
I am fine with making this a subroutine, it cleans the namelist code up a tiny bit.